Exercise_B/index.css

```diff
@@ -2,3 +2,36 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+
+.blue{
+    display: inline-block;
+    float: left;
+    width: 100px;
+    height: 100px;
+    background-color: blue;
+}
+
+.green{
+    width: 100px;
+    height: 100px;
```

Comment:
Perhaps could even segment out box so you don't have to rewrite code - remember the DRY principle (don't repeat yourself)